leaflet-providers.js

```diff
@@ -56,7 +56,7 @@
 				if (attr.indexOf('{attribution.') === -1) {
 					return attr;
 				}
-				return attr.replace(/\{attribution.(\w*)\}/,
+				return attr.replace(/\{attribution.(\w*)\}/g,
```

Comment:
Could you explain why you changed this replace ?

Reply:
I had to change it in order to be able to specify more than one attribution reference such as [`'{attribution.OpenMapSurfer}{attribution.OpenStreetMap}'`](https://github.com/leaflet-extras/leaflet-providers/blob/156398a2cad8661b5605e186607f5bda38471d78/leaflet-providers.js#L209). The original implementation would only replace the first one `{attribution.OpenMapSurfer}` leaving `{attribution.OpenStreetMap}` untouched.

Reply:
'k thx for the explaination @jieter any thoughts about this ?

Reply:
looks good to me!